ImperatorToCk2/src/main/java/ImperatorToCK2/Main.java

```diff
@@ -331,7 +331,7 @@ public static void main (String[] args) throws IOException
                     while (aq5 < irOwners.length) {
                         String[] owners = irOwners[aq5].split(","); 
 
-                        LOGGER.info(irOwners[aq5]+"_irOwners_"+aq2);  
+                        //LOGGER.info(irOwners[aq5]+"_irOwners_"+aq2);  
```

Comment:
I still could use these commented logging statements in case somebody has a save that causes problems/crashes. On my build, I can re-enable these logging statements when I parse the save through to get a thorough analysis on the problem. 

After diagnosing and fixing the issue, they could then be disabled again so they don't create 1,000+ line error logs for general users

Reply:
There are better ways to accomplish what you want, but fair enough for now.